CHANGELOG.md

```diff
@@ -1,8 +1,95 @@
+## 1.2.0
+
+This is a feature and improvement release.
+
+* [PCP-543](https://tickets.puppetlabs.com/browse/PCP-543) Add CA mismatch as
```

Comment:
This is tricky... it wasn't slated for release, but since it was merged and we also want to get a changelog into the tag, I guess it's going out.